generators/arduino/atsdk/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 2.27)
```

Comment:
What drove the nudge up to 2.27?

Reply:
I think a mixup, that file is no longer needed actually. I will drop it from the PR. Previously I was using cmake for external dependencies, but everything is statically available now